### PR TITLE
fix: Remove jfrog repositories

### PR DESCRIPTION
**Pull request by jan-gogo**
JFrog repository becomes unavailable, dependencies cannot be resolved which is blocking daily work.
Typically Gradle build is stuck on task:

```
> Resolve dependencies of :test_runner:runtimeClasspath
```
also is throwing:

```
Could not resolve all dependencies for configuration ':test_runner:runtimeClasspath'.
   > Could not resolve com.google.http-client:google-http-client-appengine:1.39.2.
     Required by:
         project :test_runner > com.google.cloud:google-cloud-storage:1.117.1
      > Could not resolve com.google.http-client:google-http-client-appengine:1.39.2.
         > Could not get resource 'https://dl.bintray.com/kotlin/ktor/com/google/http-client/google-http-client-appengine/1.39.2/google-http-client-appengine-1.39.2.pom'.
            > Could not HEAD 'https://dl.bintray.com/kotlin/ktor/com/google/http-client/google-http-client-appengine/1.39.2/google-http-client-appengine-1.39.2.pom'. Received status code 502 from server: Bad Gateway
```

Looks like all flank dependencies are available on the `mavenCentral`, so we can depend on it.

### CODE REVIEW

File: test_runner/build.gradle.kts [hunk 1]
```diff
@@ -224,7 +222,7 @@ dependencies {
 buildscript {
     repositories {
         mavenLocal()
-        jcenter()
+        // jcenter()
```
Comment (Sloox) [inline test_runner/build.gradle.kts:225]: should we not remove this completely?

**Review by Sloox — APPROVED** [review]
:+1:

**Comment by Sloox** [comment]
@flank-it

File: test_runner/build.gradle.kts (hunk 1 above)
Comment (jan-gogo) [reply test_runner/build.gradle.kts:225]:
```suggestion
```

**Pull request merged**